torch/csrc/Exceptions.h

```diff
@@ -69,45 +68,19 @@
     }
 
 #define END_HANDLE_TH_ERRORS_PYBIND                                      \
-    }                                                                    \
-    catch (py::error_already_set & e) {                                  \
-      /* Unpack already stored error to be detectable by warning code */ \
-      e.restore();                                                       \
-      throw;                                                             \
-    }                                                                    \
-    catch (py::builtin_exception & e) {                                  \
-      /* Unpack already stored error to be detectable by warning code */ \
-      e.set_error();                                                     \
-      throw;                                                             \
-    }                                                                    \
-    catch (torch::jit::JITException & e) {                               \
-      /* Special case for JITException that are explicitly unpacked by */\
-      /* pybind. Set a temporary python error to be detectable by */     \
-      /* warning code */                                                 \
-      PyErr_SetString(PyExc_RuntimeError, "JITException");               \
-      throw;                                                             \
-    }                                                                    \
-    CATCH_TH_ERRORS(throw)                                               \
   }                                                                      \
   catch (py::error_already_set & e) {                                    \
-    /* Repack already stored error */                                    \
-    throw py::error_already_set();                                       \
+    throw;                                                               \
   }                                                                      \
   catch (py::builtin_exception & e) {                                    \
-    /* Repack already stored error */                                    \
-    throw py::error_already_set();                                       \
+    throw;                                                               \
   }                                                                      \
   catch (torch::jit::JITException & e) {                                 \
-    /* Special case for JITException that are explicitly unpacked by */  \
```

Comment:
Why are these no-ops now? Is the `processErrorMsg` missing here?

Reply:
Previously the python error had to be set before entering `torch::PyWarningHandler`'s destructor. You can see it above: `e.restore()` and `e.set_error()`. `processErrorMsg` on the other hand is only used in `CATCH_TH_ERRORS`.

Reply:
Ho my bad. I though that the JITExeption message was supposed to go through `processErrorMsg` but it does not seem to be the case.